ihmc-perception/src/test/java/us/ihmc/perception/heightmap/HeightMapMessageToolsTest.java

```diff
@@ -26,23 +61,23 @@ public void testSpeedUnpackingHeightMapMessage()
 
       int centerIndex = HeightMapTools.computeCenterIndex(widthInMeters, cellResolution);
       int cellsPerAxis = 2 * centerIndex + 1;
-      Mat heightMap = new Mat(cellsPerAxis, cellsPerAxis, opencv_core.CV_32FC1);
+      Mat heightMap = new Mat(cellsPerAxis, cellsPerAxis, opencv_core.CV_16UC1);
       for (int i = 0; i < cellsPerAxis; i++)
       {
          for (int j = 0; j < cellsPerAxis; j++)
          {
-            heightMap.ptr(i, j).putFloat(1.0f);
+            heightMap.ptr(i, j).putInt(1);
```

Comment:
Shouldn't it be this?
```suggestion
            heightMap.ptr(i, j).putShort(1);
```

Reply:
If the message was shorts then yes, its still ints so technical this is correct